v5/broker/proxy/README.md

```diff
@@ -9,11 +9,11 @@ Here's a simple usage guide
 ### Run Proxy
 
 ```
-# install micro
-go get github.com/micro/micro
+# install mu
+go install github.com/micro/micro/v5/mu@latest
 
 # run proxy
-micro proxy
+mu proxy
```

Comment:
Oh, I'm not even sure this will work anymore because we totally scrapped everything...lol